client/package.json

```diff
@@ -827,7 +824,7 @@
     "test:grammar": "vscode-tmgrammar-test ./test/grammars/test-cases/*.bb",
     "snap-grammar": "vscode-tmgrammar-snap ./test/grammars/snaps/*.bb -u",
     "vscode:prepublish": "npm install --omit=dev && cd ../server && npm install --omit=dev && npm run installServer",
-    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64"
+    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web"
```

Comment:
Have you tested the generated package on a web session? (not just the debug launch)

Reply:
As far as I know, it is impossible since the web doesn't support using the CLI to install the extension.

Reply:
Can't it be done through the extension panel and give it a .vsix?

Reply:
Even https://vscode.dev/ doesn't have the ability to install a package. We'll have to rely on our experience that packaging used to work as we want in the 2.0.0 universal release